products/windows.md

```diff
@@ -176,6 +176,8 @@ releases:
 
 ---
 
+> [Microsoft Windows](https://www.microsoft.com/windows) is the operating system developed by Microsoft Corporation to run on personal computers.
```

Comment:
Do we need to add `to run on personal computers` part ? because there are some devices which are not computers [such as iot devices or some old tablets and kiosks ] and some non personal computers so it might be much better to
keep this part as `...  the operating system developed by Microsoft Corporation`

Reply:
@captn3m0 ^^